Sources/Commands/SwiftTool.swift

```diff
@@ -44,6 +43,10 @@ import var TSCUtility.verbosity
 
 typealias Diagnostic = Basics.Diagnostic
 
+// TODO: Refactor in upcoming PRs
+import Build
+import XCBuildSupport
+
```

Comment:
This will be removed when extracting `SwiftTool` into its own module in a separate PR.